.github/workflows/publish-artifacts.yml

```diff
@@ -27,4 +27,6 @@ jobs:
           VERSION=$(grep defaultBeatVersion version/version.go | cut -f2 -d "\"")
           FILENAME=$(date +"cloudformation-$VERSION-%Y-%m-%d-%H-%M-%S.yml")
           echo "Uploading $FILENAME to S3"
+          # TODO: Remove upload to vuln-mgmt/ after integration starts using new link
           aws s3 cp deploy/cloudformation/elastic-agent-ec2.yml s3://elastic-cspm-cft/vuln-mgmt/$FILENAME
```

Comment:
You can already remove that, the file is still in s3. No need to have multiple uploads.

Reply:
https://github.com/elastic/cloudbeat/pull/978